zephyr/CMakeLists.txt

```diff
@@ -564,6 +564,7 @@ zephyr_library_sources(
 	${SOF_AUDIO_PATH}/pipeline/pipeline-schedule.c
 	${SOF_AUDIO_PATH}/pipeline/pipeline-stream.c
 	${SOF_AUDIO_PATH}/pipeline/pipeline-xrun.c
+	${SOF_AUDIO_PATH}/pipeline/chain_dma.c
```

Comment:
Will need a Kconfig for this as it's probably and Intel only feature. We can make it default = y for TGL, ADL and MTL though.

Reply:
Done